numpy/__init__.pyi

```diff
@@ -1231,3 +1231,114 @@ def amin(
     initial: _NumberLike = ...,
     where: _ArrayLikeBool = ...,
 ) -> Union[number, ndarray]: ...
+
+# TODO: `np.prod()``: For object arrays `initial` does not necasarily
```

Comment:
```suggestion
# TODO: `np.prod()``: For object arrays `initial` does not necessarily
```

Reply:
Fixed in https://github.com/numpy/numpy/pull/16729/commits/c2e27c26c00993660c87944a271e5e11846f0285.